
The libpostprocess altivec parts do not all compile. So disable those for now.

  - Rene Rebe <rene@exactcode.de>

diff -ur MPlayer-1.0pre5-orig/postproc/swscale.c MPlayer-1.0pre5/postproc/swscale.c
--- MPlayer-1.0pre5-orig/postproc/swscale.c	2004-07-06 15:52:56.000000000 +0200
+++ MPlayer-1.0pre5/postproc/swscale.c	2004-10-19 01:44:35.734459896 +0200
@@ -61,6 +61,10 @@
 #else
 #include <stdlib.h>
 #endif
+
+#undef HAVE_ALTIVEC_H
+#undef HAVE_ALTIVEC
+
 #ifdef HAVE_ALTIVEC_H
 #include <altivec.h>
 #endif
@@ -1344,9 +1348,6 @@
 
 #else
 #ifdef ARCH_POWERPC
-	if(flags & SWS_CPU_CAPS_ALTIVEC)
-	  return swScale_altivec;
-	else
 	  return swScale_C;
 #endif
 	return swScale_C;
diff -ur MPlayer-1.0pre5-orig/postproc/swscale_internal.h MPlayer-1.0pre5/postproc/swscale_internal.h
--- MPlayer-1.0pre5-orig/postproc/swscale_internal.h	2004-06-27 02:07:15.000000000 +0200
+++ MPlayer-1.0pre5/postproc/swscale_internal.h	2004-10-19 01:43:19.744012200 +0200
@@ -21,6 +21,10 @@
 
 #include "../mp_msg.h"
 
+#undef HAVE_ALTIVEC_H
+#undef HAVE_ALTIVEC
+
+
 #define MSG_WARN(args...) mp_msg(MSGT_SWS,MSGL_WARN, ##args )
 #define MSG_FATAL(args...) mp_msg(MSGT_SWS,MSGL_FATAL, ##args )
 #define MSG_ERR(args...) mp_msg(MSGT_SWS,MSGL_ERR, ##args )
--- MPlayer-1.0pre5/postproc/Makefile.vanilla	2004-06-27 02:07:15.000000000 +0200
+++ MPlayer-1.0pre5/postproc/Makefile	2004-11-18 14:45:47.387899512 +0100
@@ -7,9 +7,9 @@
 
 SWSOBJS=$(SWSSRCS:.c=.o)
 
-ifeq ($(TARGET_ALTIVEC),yes)
-SWSOBJS +=  yuv2rgb_altivec.o
-endif
+#ifeq ($(TARGET_ALTIVEC),yes)
+#SWSOBJS +=  yuv2rgb_altivec.o
+#endif
 
 CFLAGS  = $(OPTFLAGS) $(MLIB_INC) -I. -I.. $(EXTRA_INC)
 # -I/usr/X11R6/include/
